client/src/components/App/Content/LandingPage/InvestigationTable/useInvestigationTable.ts

```diff
@@ -7,281 +7,387 @@ import { setIsLoading } from 'redux/IsLoading/isLoadingActionCreators';
 
 import User from 'models/User';
 import theme from 'styles/theme';
+import County from 'models/County';
+import logger from 'logger/logger';
 import Investigator from 'models/Investigator';
+import {Service, Severity} from 'models/Logger';
 import { timeout } from 'Utils/Timeout/Timeout';
 import StoreStateType from 'redux/storeStateType';
 import axios, { activateIsLoading } from 'Utils/axios';
 import { initialUserState } from 'redux/User/userReducer';
 import InvestigationTableRow from 'models/InvestigationTableRow';
 import InvestigationStatus from 'models/enums/InvestigationStatus';
-import { setEpidemiologyNum, setCantReachInvestigated, setAxiosInterceptorId } from 'redux/Investigation/investigationActionCreators';
```

Comment:
i think that maya sason added this function yesterday to solve bug in updating interceptorId, are you sure the it have no use?